core/ui/src/main/kotlin/nl/q42/template/ui/theme/PreviewAnnotations.kt

```diff
@@ -1,17 +1,16 @@
 package nl.q42.template.ui.theme
 
-import android.content.res.Configuration.UI_MODE_NIGHT_NO
 import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import android.content.res.Configuration.UI_MODE_TYPE_NORMAL
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewFontScale
 
-@Preview(
-    name = "Dark",
-    showBackground = true,
-    uiMode = UI_MODE_NIGHT_YES
+@Retention(AnnotationRetention.BINARY)
+@Target(
+    AnnotationTarget.ANNOTATION_CLASS,
+    AnnotationTarget.FUNCTION
 )
-@Preview(
-    name = "Light",
-    showBackground = true,
-    uiMode = UI_MODE_NIGHT_NO
-)
-annotation class PreviewLightDark
\ No newline at end of file
+@PreviewFontScale
```

Comment:
This stacking of previews generates 9 Previews of which 7 are fontscales. Perhaps we can limit it to a few extremes? and combine some? Like a single preview ?

Like you could have 85%, 100%, 150%, 200%
And then a single extra preview that is 

```
@Preview(name = "DarkFrenchSmall", uiMode = UI_MODE_NIGHT_YES or UI_MODE_TYPE_NORMAL, showBackground = true, locale = "fr", device = "id:small_phone")
```

That would bring the amount of previews down from 9 to 5

Also very small nit: I was thinking the most common primary and secondary languaghe would be en + nl, but I understand you may have picked french because it is a language with long strings?